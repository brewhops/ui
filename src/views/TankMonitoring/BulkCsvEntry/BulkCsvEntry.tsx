import React from 'react';

const BulkCsvEntry: React.FC = () => {
  return (<div>Bulk Entry</div>)
}

export default BulkCsvEntry;