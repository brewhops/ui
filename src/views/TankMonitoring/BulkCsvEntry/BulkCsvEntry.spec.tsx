import React from 'react';
import BulkCsvEntry from './BulkCsvEntry';
import { shallow } from 'enzyme';

describe('src/pages/TankMonitoring/BulkCsvEntry/BulkCsvEntry.tsx', () => {
  it('renders the BulkCsvEntry component', () => {
    expect(shallow(<BulkCsvEntry/>)).toMatchSnapshot();
  })
});