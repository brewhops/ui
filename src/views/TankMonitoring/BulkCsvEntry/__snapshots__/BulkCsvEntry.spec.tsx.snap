// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/pages/TankMonitoring/BulkCsvEntry/BulkCsvEntry.tsx renders the BulkCsvEntry component 1`] = `ShallowWrapper {}`;
